db/migrations/0003_alter_player_guild_alter_player_race.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-11-04 18:30
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_player_delete_playermodel'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.OneToOneField(null=True, on_delete=django.db.models.deletion.SET_NULL, to='db.guild'),
+        ),
+        migrations.AlterField(
+            model_name='player',
+            name='race',
+            field=models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to='db.race'),

```

Comment:
The 'race' field has been changed to a OneToOneField. This implies a one-to-one relationship between Player and Race. Verify that this change is consistent with your application's requirements.